src/ray/gcs/tables.h

```diff
@@ -27,6 +27,10 @@ class RedisContext;
 
 class AsyncGcsClient;
 
+/// Specifies whether commands issued to a table should be regular or chain-replicated
+/// (when available).
+enum class CommandType { kRegular, kChain };
```

Comment:
Thank you for using enum class! We should be doing this everwhere.

Reply:
Ack & +1